docs/general/builtins/aggregation.rst

```diff
@@ -169,6 +169,52 @@ array.
     +---------------------+
     SELECT 1 row in set (... sec)
 
+
+Example: Aggregate table names to their schema::
+
+
+    cr> select
+    ...    table_schema,
+    ...    array_agg(table_name) as tables
+    ... from information_schema.tables
+    ... group by table_schema;
+    +---------------------+-----------------------------------------+
```

Comment:
How is this different from the first example? It doesn't really show anything extra, does it?

Reply:
In my head this first example prepares the readers mind into the second example, technically it's the first as the already existing example, I'm fine to leave this out.